fpdf/html.py

```diff
@@ -280,15 +288,48 @@ def handle_data(self, data):
                 align=self.align[0].upper(),
                 link=self.href,
             )
+        elif self.pre_formatted:  # for pre blocks
+            self.pdf.write(self.h, data)
+
+        elif (
+            self.follows_fmt_tag and not self.follows_trailing_space
+        ):  # don't trim leading whitespace if following a format tag with no trailing whitespace
```

Comment:
`black` can be annoying with how it formats inline comments.
Maybe it would be clearer to just put this comment on an empty line after the `elif` ?